Packs/Core/Playbooks/playbook-Excessive_User_Account_Lockouts_README.md

```diff
@@ -0,0 +1,67 @@
+This playbook is designed to handle the following alerts:
+
+- Excessive user account lockouts
+- Excessive account lockouts on suspicious users
+- Excessive user account lockouts from a suspicious source
+
+The playbook investigates and responds to excessive user account lockout alerts. It gathers information about the alert, enriches relevant host data, and analyzes event patterns. This analysis helps distinguish between benign lockouts and lockouts caused by brute-force or password spray attacks.
+
+Playbook Stages:
+
+Triage:
+
+- The playbook enriches the alert with details about the lockout events.
+
+Investigation:
+
+- The playbook analyzes the lockout event timestamps to detect patterns, checks for related medium severity brute-force alerts in the incident, and retrieves the Risk Score for the Caller Computer that caused the lockouts.
+
+Containment:
+
+- With analyst approval, the playbook can isolate the endpoint (either the Caller Computer or the target host) if it's determined to be a true positive and not a server.
+
+Requirements:
+
+- For response actions, the following integrations are required: Core - IR.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
```

Comment:
it appears the playbook does not have any dependencies, so this section can be removed